ci_feature/main.sh

```diff
@@ -19,6 +19,9 @@ mkdir -p /var/opt/microsoft/docker-cimprov/state
   #sudo setfacl -m user:omsagent:rw /var/run/host/docker.sock
 #fi
 
+# add permissions for omsagent user to access azure.json
+sudo setfacl -m user:omsagent:rw /etc/kubernetes/azure.json
```

Comment:
why do you need write perms on azure.json ?

Reply:
changed to r